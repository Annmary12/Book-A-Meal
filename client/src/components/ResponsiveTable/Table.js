import React from 'react';
import PropTypes from 'prop-types';
import styles from './responsive-table.scss';

/**
 *
 * @param {Object} props
 *
 * @returns {JSX} React JSX
 */
const Table = props => (
  <div className={styles.table}>
    {props.children}
  </div>
);

Table.propTypes = {
  children: PropTypes.node.isRequired
};

export default Table;
