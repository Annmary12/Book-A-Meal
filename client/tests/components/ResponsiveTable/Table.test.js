import React from 'react';
import Table from '../../../src/components/ResponsiveTable/Table';

describe('ResponsiveTable Table component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Table />);

    expect(wrapper).toMatchSnapshot();
  })
})