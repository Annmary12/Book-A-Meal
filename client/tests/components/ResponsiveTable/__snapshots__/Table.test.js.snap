// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResponsiveTable Table component should match snapshot 1`] = `
<div
  className="table"
/>
`;
